conanfile.py

```diff
@@ -17,8 +17,13 @@ def requirements(self):
         if self.options.build_unittest:
             self.requires.add('doctest/2.3.4@bincrafters/stable')
 
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.install()
```

Comment:
Comands of conan to install the `cmake` targets, no need for the build phase.

Reply:
Better move this to the `def pacakge()` command

Reply:
Hi @danimtb, commands moved to `def package()`.